### ISSUE TITLE
NTR: Auricular inflammation (Ears)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7812]
New term request Auricular inflammation (sundel_181213195413): Ear auricles redness swelling; helix, lobule, tragus  UMLS: 2076570

**Comment by pnrobinson** [comment]
@MickeySegal
The following term is very close (it can also include the external ear canal), but unless there is a reason not to do so, probably we should add Auricular inflammation as a synonym
[Otitis externa HP:0410017](https://hpo.jax.org/app/browse/term/HP:0410017)

**Comment by MickeySegal** [comment]
https://hpo.jax.org/app/browse/term/HP:0410017 has Inflammation or infection of the external auditory canal (EAC), the auricle, or both.

What we want is the auricle inflammation part, https://hpo.jax.org/app/browse/term/HP:0200047, which should perhaps also be a child of the above.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000772
- label: Auriculitis
- parents: [Abnormal pinna morphology (HP:0000377)](https://hpo.jax.org/app/browse/term/HP:0000377)
- pmid: 
- def: Inflammation of the skin and soft tissues of the outer ear.
- comment: In contrast to auricular chondritis, auriculitis does not specifically affect the cartilage of the ear.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7812
- synonyms: Auricular inflammation(NONE)
